### PR TITLE
Recaptcha Config Changes

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the changes for Recaptcha Config QA bug fix.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js [hunk 1]
```diff
@@ -53,15 +53,17 @@ function validateContactFrom(e) {
...
-    if(validateRecaptcha() == false) {
-        ErrorMsgElement.style.visibility = "visible";
-        ErrorMsgElement.innerHTML = '<span>please verify you are human!<span>';
-         isValid = false;
-        return false;
-    } else {
-        ErrorMsgElement.style.visibility = "hidden";
-        ErrorMsgElement.innerHTML = '';
+    const ErrorMsgElement = document.getElementById('cmp-contactus-errormsg');
+    if(value){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js]: @nitheshkumar-hcl : what value is this ? please add a relevant variable name

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
+var value = $('#recaptcha').data('custom-property');
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js]: @nitheshkumar-hcl : Pl add relevant variable name here

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js (hunk 1 above)
Comment (rishabh-behl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js]: also put variable declaration in contactus.js

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+if(value){
 function validateRecaptcha(){
+
     var response = grecaptcha.getResponse();
     if(response.length == 0)
     {
         return false;
     }
-    return true;
+
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js:10]: @nitheshkumar-hcl : remove extra space

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/js/signin.js [hunk 4]
```diff
@@ -155,12 +155,14 @@ const handleHttpServerRequestJson = function (url, formdata) {
 	  document.login_form.password.focus();
 	  return false;
 	}
-	if(validateRecaptcha() == false && failedLoginCounter > 1) {
-		ErrorMsgElm.style.visibility = "visible";
-		ErrorMsgElm.innerHTML = "<span>please verify you are human!<span>";
-		  //errorhtml = '<span>please verify you are humann!<span>';
-		  //displayError(errorhtml, 'terms');
-		return false;
+	if(value){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/js/signin.js]: @nitheshkumar-hcl : same as above comment

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signup/clientlibs/js/signup.js [hunk 5]
```diff
@@ -133,11 +133,13 @@ function displayError(errorhtml, formElmName) {
 		  displayError(errorhtml, 'terms');
 		  return false;
 	  }
-	if(validateRecaptcha() == false) {
-          errorhtml = '<span>please verify you are human!<span>';
-          displayError(errorhtml, 'terms');
-          return false;
-      }
+	  if(value){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signup/clientlibs/js/signup.js]: @nitheshkumar-hcl : same as above comment

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@nitheshkumar-hcl : pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js (hunk 1 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/contactus/clientlibs/js/contactus.js]: Variable Name Changed as per the review comments.variable declaration for contact us.js is provided in recaptcha.js

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js (hunk 2 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js]: Changed as per the review comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js (hunk 3 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/recaptcha/clientlibs/js/recaptcha.js:10]: removed as per the review comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/js/signin.js (hunk 4 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signin/clientlibs/js/signin.js]: Changed as per the review comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signup/clientlibs/js/signup.js (hunk 5 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signup/clientlibs/js/signup.js]: Changed as per the review comments

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**